src/components/home/Service.js

```diff
@@ -1,7 +1,28 @@
+"use client";
 import React from "react";
-
+import { HiCodeBracket } from "react-icons/hi2";
 const Service = () => {
-  return <div>Service</div>;
+  return (
+    <div className="bg-winc-yellow w-96 h-96 rounded-lg  flex items-center justify-center">
+      <div className="flex flex-column items-center w-11/12">
```

Comment:
what is this div for? also it is flex-col. Try not to use extra divs

Reply:
Removed the extra div and chnaged it